src/DynamoCore/Graph/Annotations/GroupStyleItemEntry.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Dynamo.Graph.Annotations
+{
+    /// <summary>
+    /// This class represent a GroupStyle item that will be shown in the Group ContextMenu (basically a MenuItem)
+    /// </summary>
+    internal class GroupStyleItemEntry : GroupStyleBase
+    {
+        private string hexColorString;
+        private string textContent;
+        private bool isChecked = false;
+        private bool isDefault = false;
+
+        public string HexColorString
```

Comment:
Add summary to public fields.

Reply:
@reddyashish added summary comments to public properties in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12667/commits/b9e445eaa56468337706ac88105d9b921912a90d